L1Trigger/TrackFindingTracklet/src/AllProjectionsMemory.cc

```diff
@@ -0,0 +1,49 @@
+#include "L1Trigger/TrackFindingTracklet/interface/AllProjectionsMemory.h"
+#include "L1Trigger/TrackFindingTracklet/interface/Tracklet.h"
+#include "L1Trigger/TrackFindingTracklet/interface/Settings.h"
+
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+
+using namespace Trklet;
+using namespace std;
+
+AllProjectionsMemory::AllProjectionsMemory(string name, const Settings* const settings, unsigned int iSector)
+    : MemoryBase(name, settings, iSector) {
+  initLayerDisk(3, layer_, disk_);
+}
+
+void AllProjectionsMemory::writeAP(bool first) {
+  std::string fname = "../data/MemPrints/TrackletProjections/AllProj_";
```

Comment:
Please do not mix operations on a `string` and on a `stringstream`; rather, just use the `stringstream` to format the whole string.

The whole chunk cold become
```c++
std::ostringstream oss;
oss << "../data/MemPrints/TrackletProjections/AllProj_"
    << getName()
    << "_"
    << std::setfill('0') << std::setw(2) << (iSector_ + 1)
    << ".dat";
auto const& fname = oss.str();
```

[suggestion] or use `boost::format`

Reply:
implemented as suggested